MyApp/Homework-Assignment-SevenA.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this is derived data. This is for your machine only. This file shouldn't be in the PR

Reply:
Can I just delete this file straight from my pull request and then commit it to the feature branch?

Reply:
I deleted the file